Source: Anders Ossowicki
Upstream: No
Reason: We do not guarantee that a syslogger exists since it's not a strict requirement for building and installing the module. Also, silly people are silly!

From cbc9da160f17ab28e979d232418a1e68daf935b0 Mon Sep 17 00:00:00 2001
From: Anders Ossowicki <arkanoid@exherbo.org>
Date: Sun, 9 Aug 2009 02:08:35 +0200
Subject: [PATCH] Remove Syslog tests

---
 cpan/Sys-Syslog/t/syslog.t |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/cpan/Sys-Syslog/t/syslog.t b/cpan/Sys-Syslog/t/syslog.t
index 56a83c7..5d83c61 100444
--- a/cpan/Sys-Syslog/t/syslog.t
+++ b/cpan/Sys-Syslog/t/syslog.t
@@ -23,6 +23,8 @@ use warnings qw(closure deprecated exiting glob io misc numeric once overflow
 # must manually disable warnings
 $^W = 0 if $] < 5.006;
 
+plan skip_all => "[Exherbo] We do not enforce the existence of a system logger";
+
 my $is_Win32  = $^O =~ /win32/i;
 my $is_Cygwin = $^O =~ /cygwin/i;
 
-- 
1.6.4

